docs/extras/autorcm.md

```diff
@@ -12,29 +12,29 @@
 	- When completely discharged, your Switch will take a very long time to charge while in RCM. To solve this, charge the console for approximately twenty minutes before booting into Hekate and selecting a boot option. After it's done booting, the console will now charge at a normal speed with a battery icon in the corner.
-	- When a system update is completed while stock firmware is running, AutoRCM becomes disabled so you need to turn it on again. AutoRCM can be preserved by updating while CFW is active.
+	- When a system update is completed while stock firmware is running, autoRCM becomes disabled so you need to turn it on again. autoRCM can be preserved by updating while CFW is active.
 
-!!! tip "Advantages of AutoRCM"
+!!! tip "Advantages of autoRCM"
 	AutoRCM can be used for good as well:
 
 	- If you were having trouble getting into RCM before (i.e you were using the tinfoil method), now you can get into RCM without any hassle.
-	- If you want to keep fuses unburned, you can do so with AutoRCM, as the Switch never gets the chance to burn them. Nintendo's bootloaders burn the fuses, but those bootloaders cannot be used if AutoRCM is enabled.
-		Note: updating the console while booted into Hekate's "stock" configuration will clear AutoRCM on the next reboot. Please update while using custom firmware, or be sure to use a jig after the console reboots.
+	- If you want to keep fuses unburned, you can do so with autoRCM, as the Switch never gets the chance to burn them. Nintendo's bootloaders burn the fuses, but those bootloaders cannot be used if autoRCM is enabled.
+		Note: updating the console while booted into Hekate's "stock" configuration will clear autoRCM on the next reboot. Please update while using custom firmware, or be sure to use a jig after the console reboots.
 	- It's easier to inject a payload using a payload injector, as using your jig and booting into RCM manually is no longer required.
 
 !!! tip "Other information..."
 	- This method corrupts a single byte in both your BOOT0 and your BOOT1 partitions. This is why a backup of those is recommended.
-  - This software version of AutoRCM can almost always be reversed, so don't freak out if it doesn't work out the way you'd like it to.
+  - This software version of autoRCM can almost always be reversed, so don't freak out if it doesn't work out the way you'd like it to.
 
-If, despite all of the warnings above, you still wish to install AutoRCM, and understand the risks, do the following:
+If, despite all of the warnings above, you still wish to install autoRCM, and understand the risks, do the following:
 
 1. Boot Hekate
 2. Go to `Tools`
-3. At the bottom of the page, tap `Arch Bit • AutoRCM • Touch • Pkg1/2`
-4. Then tap `AutoRCM`. It will show a little `ON` next to the option once you have done this.
+3. At the bottom of the page, tap `Arch Bit • autoRCM • Touch • Pkg1/2`
```

Comment:
It should be `Arch bit • RCM • Touch • Pkg1/2`, and keep the a capitalized (Then tap `AutoRCM`)